.phan/config.php

```diff
@@ -1,70 +1,87 @@
 <?php
 
 return [
-	"backward_compatibility_checks" => true,
-	// The docs on quick_mode at
-	// https://github.com/etsy/phan/wiki/Incrementally-Strengthening-Analysis
-	// don't seem reasonable. They claim that quick_mode=true add more errors.
-	// This is false on the assumption that that's a typo.
-	// It doesn't seem to have any effect on LORIS's codebase anyways.
-	"quick_mode" => false,
-	"analyze_signature_compatibility" => true,
-	// FIXME: allow_missing_properties should be false, but there's
-	// too many other things to fix first.
-	"allow_missing_properties" => true,
-	"null_casts_as_any_type" => false,
-	"scalar_implicit_cast" => false,
-	"ignore_undeclared_variables_in_global_scope" => false,
-	"suppress_issue_types" => [
+    "backward_compatibility_checks" => true,
+    // The docs on quick_mode at
+    // https://github.com/phan/phan/wiki/Incrementally-Strengthening-Analysis
+    // don't seem reasonable. They claim that quick_mode=true add more errors.
+    // This is false on the assumption that that's a typo.
+    // It doesn't seem to have any effect on LORIS's codebase anyways.
+    "quick_mode" => false,
+    "analyze_signature_compatibility" => true,
+    // The default severity level for phan is 5. After removing all the
+    // suppressed rules, we should consider reducing this value to detect more
+    // suspicious code.
+    "minimum_severity" => 5,
+    // TODO: Update this value to the minimum required by LORIS.
+    "target_php_version" => "7.3",
```

Comment:
This should not be explicitly set, or Travis won't be testing the right version.